db/index.js

```diff
@@ -1,80 +1,27 @@
-var mongoose = require ('mongoose');
-var faker = require ('faker');
+const pgp = require('pg-promise')();
+var dbDetails = require('../config.js');
+let username = dbDetails.db.username;
+let password = dbDetails.db.password;
+let host = dbDetails.host;
+var db = pgp(`postgres://${username}:${password}@${host}:5432/media_player`);
```

Comment:
wow this is interesting and much more straightforward than the way I was trying to seed my db - I'm going to try this too :) very similar to mysql!

Reply:
Very similar!